languages/en.ini

```diff
@@ -1221,6 +1221,8 @@ renew_success = "Renewal Successful"
 renew_success_summary = "Successfully renewed {count, plural, =1 {1 item} other {# items}}."
 Renewed = "Renewed"
 Request full text = "Request full text"
+request_group_fulfillment_method_hold_shelf = "Pick up at the library"
+request_group_fulfillment_method_delivery = "Delivery to your location"
```

Comment:
I could just shorten the keys to "request_group_hold_shelf" etc.  But I think it's valuable to make clear that these keys are specifically from the hard-coded fulfillment methods, unlike other request group keys that may be user-configurable.

As far as the (default) values, I welcome input.  It could be as simple as "Pick up" and "Delivery".